d2core/d2gui/widget.go

```diff
@@ -44,6 +49,23 @@ func (w *widgetBase) SetPosition(x, y int) {
 	w.y = y
 }
 
+func (w *widgetBase) GetPosition() (int, int) {
+	return w.x, w.y
+}
+
+func (w *widgetBase) GetOffset() (int, int) {
+	return w.offsetX, w.offsetY
+}
+
+func (w *widgetBase) getOffset() (int, int) {
```

Comment:
Why is this duplicated as an unexported method?

Reply:
Good question. I'm looking into that last bug, I'll remove this.

Reply:
At first I wanted to keep the interface methods unexported but you are right, it looks like an unnecessary duplication => I only kept the exported one.